schema.dbml

```diff
@@ -510,6 +511,79 @@ Enum assay_enum {
   "MLV pseudovirus"
 }
 
+Table assay as AS {
+  ref_name varchar
+  assay_name assay_enum
+  upper_titer_limit int
+  lower_titer_limit int
+}
+
+Table fold as FD {
+  ref_name varchar
+  rx_name varchar
+  control_iso_name varchar [ref: > I.iso_name]
+  iso_name varchar [ref: > I.iso_name]
+  section varchar [
+    note: 'The cited section in the reference; e.g. Figure 1, table 1 or page 1, para 4'
+  ]
+  fold_cmp numeric_cmp_enum [
+    default: "=",
+    note: '"=", ">", or "<"'
+  ]
+  fold numeric(10,1) [
+    note: 'Fold change comparing to wild type'
+  ]
+  inhibition_pcnt numeric(2,0) [
+    note: 'Percentage of inhibition (e.g. IC50, IC90); 0 means unknown or is pending for entering'
+  ]
+  resistance_level resistance_level_enum [
+    note: 'Qualitative result when quantitative result (fold change) is not available'
+  ]
+  ineffective ineffective_enum [
+    note: 'Rx has no effect on control sequence or experimental sequence'
+  ]
+  cumulative_count int [
+    default: 1,
+    note: 'Cumulative count of matching results (when multiple)'
+  ]
+  assay_name assay_enum
+  date_added date [not null]
+  indexes {
+    (rx_name, control_iso_name, iso_name, ref_name, inhibition_pcnt) [pk]
+  }
+}
+
+Table titer as TTR {
```

Comment:
Can we rename the table to "rx\_potency" and change column name "titer" to "potency\_type" and "potency\_value"? The "potency\_type" can be titer, EC50, EC90, etc.